package.json

```diff
@@ -136,11 +138,13 @@
     "prom-client": "^12.0.0",
     "proxy-agent": "^3.1.1",
     "querystringify": "^2.1.1",
+    "react-use": "^15.3.4",
```

Comment:
this library is just being used for a nice `useDebounce` (imported independently) but if we don't want to add this dep i'm happy to write my own.

Reply:
These dependencies are great, but can we move them to `devDependencies`? They aren't used by the server so we don't need to include it in `dependencies`.